angular.json

```diff
@@ -124,6 +124,10 @@
           }
         }
       }
-    }},
-  "defaultProject": "angular-v8"
+    }
+  },
+  "defaultProject": "angular-v8",
+  "cli": {
```

Comment:
Just curious what may have caused this to be added.  Did you specifically add it or maybe the cli did when you created a component?